client/lib/wp/sync-handler/index.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Module dependencies
+ */
+import config from 'config';
+import localforage from 'localforage';
+import Hashes from 'jshashes';
+import { isWhitelisted } from './whitelist-handler';
+
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:sync-handler' );
+
+// defaults
+const defaults = config( 'sync-handler-defaults' );
+defaults.driver = localforage[ defaults.driver ];
```

Comment:
yup :+1: 

We can stick with the config defaults though, if we imagine that different environments will have different setup values.